pages/content/amp-dev/documentation/components/reference/amp-accordion@es.md

```diff
@@ -60,14 +60,26 @@ El componente `amp-accordion` permite mostrar secciones de contenido que se pued
 
 En este ejemplo, se muestran tres secciones; la tercera se expande cuando se carga la página.   Además, hemos añadido `disable-session-states` para que no se mantenga el estado expandido u oculto de las secciones.
 
-<!--ejemplo insertado - se muestra en ampproject.org -->
-
-<div>
-  <amp-iframe height="395" src="https://ampproject-b5f4c.firebaseapp.com/examples/ampaccordion.basic.embed.html" layout="fixed-height" sandbox="allow-scripts allow-forms allow-same-origin" resizable="">
-    <div aria-label="Show more" overflow="" tabindex="0" role="button">Mostrar código completo</div>
-    <div placeholder=""></div>
-  </amp-iframe>
-</div>
+[example preview="inline" playground="true" imports="amp-accordion"]
```

Comment:
it'd be better to filter the whole section for amp-email. The explanation before doesn't apply to email either.

Reply:
This example is the same also for the en version, so this change does not apply for the translations only and we would also need a PR for the amphtml project.

Reply:
I think only the disable-session-states is not valid for email. Otherwise this is the only code sample and therefore should stay in the page.

Reply:
There's a [filter syntax on markdown level](https://github.com/ampproject/amp.dev/blob/future/contributing/documentation.md#format-filtered-paragraphs) we should use instead to filter code sample and explanation. 

Can you create a PR for this on ampproject/amphtml as well?

Reply:
I would prefer if those changes are done separately by someone who knows more about the component when used in emails. It is not only the example, but also the attribute description for `disable-session-states` that needs to change.

Reply:
https://github.com/ampproject/amp.dev/issues/2646